editor/filesystem_dock.cpp

```diff
@@ -899,7 +939,7 @@ void FileSystemDock::_update_file_list(bool p_keep_selection) {
 	}
 
 	Ref<Texture2D> folder_icon = (use_thumbnails) ? folder_thumbnail : get_theme_icon(SNAME("folder"), SNAME("FileDialog"));
-	const Color folder_color = get_theme_color(SNAME("folder_icon_color"), SNAME("FileDialog"));
+	Color default_folder_color = get_theme_color(SNAME("folder_icon_color"), SNAME("FileDialog"));
```

Comment:
Why did you remove `const`?

Reply:
Oops, at one point I was changing that variable further down in order to test something afaik but must've forgotten to make it `const` again.